src/en/actions.md

```diff
@@ -70,25 +70,27 @@ add-repo:
 ```
 
 !!! Note: 
-    the full schema is under the `properties` key of the root Action.  Juju
-    Actions rely on [JSON-Schema][jsonschema] for validation.  The
-    top-level keys shown for the Action (`description` and `properties`) may
-    include future additions to the feature.
+    The full schema is under the `properties` key of the root action. Actions
+    rely on [JSON-Schema][jsonschema] for validation. The top-level keys shown
+    for the action (`description` and `properties`) may include future
+    additions to the feature.
+
+### `run-action`
 
-### `juju run-action`
+Triggers an action.
 
-Trigger an action. This command takes a unit (or multiple units) as an argument
-and returns an ID for the action. The ID can be used with `juju
-show-action-output <ID>` or `juju show-action-status <ID>`.
+This command takes a unit (or multiple units) as an argument and returns an ID
```

Comment:
I feel like the /leader should be noted here as basically it's another format for specifying "which unit". While we talking about taking a unit/multiple having the leader option seems to fit here.